### PR TITLE
Feat/session readonly

### PR DESCRIPTION
**Pull request by wrjgold**
Description: 
Adds an option to invite other users to a session as viewers (read-only mode) or editors.

Changes:
Added new read only option field when creating a ShareAce instance
Sets editor binding’s read only mode to true/false depending on arguments passed into read only field

### CODE REVIEW

File: source/sharedb-ace.js [hunk 1]
```diff
@@ -68,9 +69,8 @@ class SharedbAce extends EventEmitter {
 
       this.emit('ready');
     };
-
+
```
Comment (RichDom2185) [inline source/sharedb-ace.js:72]: Minor nit: let's remove the trailing whitespace here.

**Review by RichDom2185 — COMMENTED** [review]
Hi, looks good! Thanks for working on this; will you need this package to be updated to NPM after this PR is merged?

**Comment by izruff** [comment]
We found a problem with these changes: upon leaving a view-only session, the expected behavior is that the editor returns to normal (its `readOnly` option set to `false`), but here the option remains even after destroying the instance. We missed this case during testing.

I don't think there is an easy way to enforce this from the client. Instead, we should enable/disable the editor in the frontend. Thus, this PR may not be necessary.

We'll try to update our frontend PR, and once we manage to fix the bug, we'll consider closing this PR.

**Comment by RichDom2185** [comment]
As discussed via Telegram, this PR is no longer needed to achieve the functionality.

**Pull request closed without merging**